zzzz_modular_occulus/code/modules/economy/vending.dm

```diff
@@ -171,23 +171,39 @@
 		)
 
 /obj/machinery/vending/weapon_machine
-	prices = list(
+	products = list(
+					/obj/item/weapon/reagent_containers/spray/pepper = 5,
+					/obj/item/weapon/gun/projectile/revolver/havelock = 5,
+					/obj/item/weapon/gun/projectile/revolver/saa = 5,
```

Comment:
Why does this gun need to be in a vendor at all? It doesn't seem like a standard arm like the others in here. It seems like a maint gun.